node/src/chain_spec.rs

```diff
@@ -686,7 +653,14 @@ fn foucoco_genesis(
 	start_shutdown: bool,
 ) -> foucoco_runtime::GenesisConfig {
 	fn default_pair(currency_id: CurrencyId) -> VaultCurrencyPair<CurrencyId> {
-		VaultCurrencyPair { collateral: currency_id, wrapped: TESTNET_DEFAULT_WRAPPED_CURRENCY_ID }
+		VaultCurrencyPair { collateral: currency_id, wrapped: MAINNET_USDC_CURRENCY_ID }
+	}
```

Comment:
Let's remove the `default_pair` function here as we don't use it.

Reply:
Done!